virttest/shared/cfg/base.cfg

```diff
@@ -827,6 +827,11 @@ virtinstall_extra_args = ""
 #realtime_mlock = on
 #keyboard_layout = en-us
 
+# If to move VM logs - usually located under /var/log/libvirt/qemu/
+# into the test debug directory. This can help confirm the qemu command
+# and crashes.
+store_libvirt_vm_logs = "yes"
```

Comment:
Thanks for updating the configuration parameter name @smitterl. I'm adding this comment here just to mark it as "resolved" right after.